src/taco-cli/cli/build.ts

```diff
@@ -137,7 +137,7 @@ class Build extends commands.TacoCommandBase {
         });
     }
 
-    private static build(commandData: commands.ICommandData): Q.Promise<tacoUtility.ICommandTelemetryProperties> {
+     protected runCommand(commandData: commands.ICommandData): Q.Promise<ICommandTelemetryProperties> {
```

Comment:
yes. I would love to have that. But I would let the code owner drive that cleanup..